src/bindings/interface/importer.i

```diff
@@ -0,0 +1,58 @@
+%module(package="libcellml") importer
+
+#define LIBCELLML_EXPORT
+
+%import "createconstructor.i"
+%import "logger.i"
+%import "types.i"
+
+%feature("docstring") libcellml::Importer
+"Imports CellML objects.";
+
+%feature("docstring") libcellml::Importer::resolveImports
+"Resolves all imports in this model.
+
+Resolves all :class:`Component` and :class:`Units` imports by loading the
+models from local disk through relative urls. The ``baseFile`` is used to
+determine the full path to the source model relative to this one.";
+
+%feature("docstring") libcellml::Importer::flatten
+"Instantiate all imported Components and Units and return a self-contained model.";
+
+%feature("docstring") libcellml::Importer::libraryCount
+"Returns the number of models loaded into the importer's library."
+
+%feature("docstring") libcellml::Importer::library
+"Returns the model instance stored at the given url in the library.
+The url string must be the absolute path to the document, including both the filename
+and the baseFile path."
+
+%feature("docstring") libcellml::Importer::addModel
+"Manually add a local model instance into the library under the url key provided.
+Returns true if the model was added, and false if the url key already exists."
```

Comment:
Replace "true"/"false" with "`True`"/"`False`".

Reply:
0ecc37d9

Reply:
FWIW, to reference a commit that has not been pushed results in plain text, not a hyperlink (even if you then push the commit; at least, in my experience).